### ISSUE TITLE
[page #100 ] - deployment YAML file not complete

### ISSUE THREAD

**Issue by tkaagent** [pinggit/kubernetes-contrail-day-one#30]
deploy-webserver-do.yaml file should be corrected; As per page #100:

[..]
In both namespaces and the default namespace, create a deployment to launch a webserver pod:

#deploy-webserver-do.yaml
apiVersion: apps/v1
- {key: app, operator: In, values: [webserver]}
^^^^^^^^^^^^^^^^^^^^^^^^

$ kubectl apply -f deploy-webserver-do.yaml -n default
deployment.extensions/webserver created

 
Recommendation:
============== 
apiVersion: apps/v1
kind: Deployment
metadata:
  name: webserver
  labels:
    app: webserver
spec:
  replicas: 1
  selector:
    matchLabels:
      app: webserver
    matchExpressions:
      - {key: app, operator: In, values: [webserver]}
  template:
    metadata:
      name: webserver
      labels:
        app: webserver
    spec:
      containers:
      - name: webserver
        image: contrailk8sdayone/contrail-webserver
        securityContext:
           privileged: true
        ports:
        - containerPort: 80

**Comment by pinggit** [comment]
that is right. it is missing in the official final PDF. it is in the raw draft though.
will fix it.
